src/main/java/com/ericsson/ei/config/CheckEIConfigurations.java

```diff
@@ -0,0 +1,95 @@
+                + "waitlist.initialDelayResend: " + env.getProperty("waitlist.initialDelayResend") + "\n"
+                + "waitlist.fixedRateResend: " + env.getProperty("waitlist.fixedRateResend") + "\n"
+                + "subscription.collection.name: " + env.getProperty("subscription.collection.name") + "\n"
+                + "subscription.collection.repeatFlagHandlerName: " + env.getProperty("subscription.collection.repeatFlagHandlerName") + "\n"
+                + "testaggregated.enabled: " + env.getProperty("testaggregated.enabled") + "\n"
+                + "threads.corePoolSize: " + env.getProperty("threads.corePoolSize") + "\n"
+                + "threads.queueCapacity: " + env.getProperty("threads.queueCapacity") + "\n"
+                + "threads.maxPoolSize: " + env.getProperty("threads.maxPoolSize") + "\n"
+                + "missedNotificationCollectionName: " + env.getProperty("missedNotificationCollectionName") + "\n"
+                + "missedNotificationDataBaseName: " + env.getProperty("missedNotificationDataBaseName") + "\n"
+                + "email.sender: " + env.getProperty("email.sender") + "\n"
+                + "email.subject: " + env.getProperty("email.subject") + "\n"
+                + "notification.failAttempt: " + env.getProperty("notification.failAttempt") + "\n"
+                + "notification.ttl.value: " + env.getProperty("notification.ttl.value") + "\n"
+                + "spring.mail.host: " + env.getProperty("spring.mail.host") + "\n"
+                + "spring.mail.port: " + env.getProperty("spring.mail.port") + "\n"
+                + "spring.mail.username: " + env.getProperty("spring.mail.username") + "\n"
+                + "spring.mail.properties.mail.smtp.auth: " + env.getProperty("spring.mail.properties.mail.smtp.auth") + "\n"
+                + "spring.mail.properties.mail.smtp.starttls.enable: " + env.getProperty("spring.mail.properties.mail.smtp.starttls.enable") + "\n"
+                + "er.url: " + env.getProperty("er.url") + "\n"
+                + "ldap.enabled: " + env.getProperty("ldap.enabled") + "\n"
+                + "ldap.url: " + env.getProperty("ldap.url") + "\n"
+                + "ldap.base.dn: " + env.getProperty("ldap.base.dn") + "\n"
+                + "ldap.username: " + env.getProperty("ldap.username") + "\n"
+                + "ldap.user.filter: " + env.getProperty("ldap.user.filter") + "\n"
+                + "logging.level.root: " + env.getProperty("logging.level.root") + "\n"
+                + "logging.level.org.springframework.web: " + env.getProperty("logging.level.org.springframework.web") + "\n"
+                + "logging.level.com.ericsson.ei: " + env.getProperty("logging.level.com.ericsson.ei") + "\n"
+                + "\nThese properties is only some of the configuraitons, more configurations can have been provided.\n");
+        
+        try {
+            new RulesHandler().readRuleFileContent(rulePath);
```

Comment:
It would be nice to extend this in the future to send the content of the file to EI frontend, so it can be visualised for the user writing subscriptions.

Reply:
We can maybe wait and write a new story for adding a new entrypoint that returns the configuraiton to the frontend web-ui, if needed. When that story has been decided to be implemented, this configuration string can be moved out to a getConfigurationAsString function.